src/microbe_stage/Microbe.cs

```diff
@@ -426,6 +426,11 @@ public override void _Process(float delta)
 
         HandleOsmoregulation(delta);
 
+        // Colony members have their movement update before organelle update,
+        // so that the movement organelles see the direction
```

Comment:
Doesn't this rely on the fact that _Process would be called first on the master microbe? Or would this work correctly in the case where the stale movement direction value is used here? I think this should at least be clarified in a comment.

Reply:
It does indeed rely on that fact. This is what this comment says, tell me what clarifications you need, for me that comment is enough.

Reply:
Normally this wouldnt be a problem i think, the player shouldnt notice the fact that the organelles movements for the miliseconds that it takes to adjust.

Reply:
Well if the parent gets the process first then its really no problem or delay. I will clarify that then

Reply:
Note that I *think* that is what happens (as Godot processes signals in scene tree order), you should verify with a debugger to confirm before writing a comment how things are.

Reply:
It really is that way no need for debugger.
https://docs.godotengine.org/en/stable/getting_started/step_by_step/scene_tree.html#tree-order

Reply:
Even if the behaviour is confirmed, remember that there could be a bug in the engine, then explaining it here is necessary to avoid future confusion. See my styleguide entry that if there's enough confusion to warrant debating about something, that's enough to require it to be commented properly in the code.

Reply:
Well anything could be a bug and be commented when thinking like that but ok.

Reply:
In unclear situations I prefer much more to just test something rather than believing documentation is right. At least after relying on the documentation I do at least a quick test of the overall operation to make sure I understood the documentation correctly / there wasn't a mistake in the documentation.